### ISSUE TITLE
Action failed

### ISSUE THREAD

**Issue by marvinm2** [marvinm2/WikiPathwaysLoader#10]
The action does not complete. The last step fails

https://github.com/marvinm2/WikiPathwaysLoader/actions/runs/5033446250

**Comment by jmillanacosta** [comment]
Yes, because of how the PR are set up in this repository a review is needed to push. Will look at it, maybe it would be a good idea to add a a test and you get notified and merge manually if it passes.
